sphinxcontrib/openapi/directive.py

```diff
@@ -48,6 +48,7 @@ class OpenApi(Directive):
         'paths': lambda s: s.split(),       # endpoints to be rendered
         'examples': directives.flag,        # render examples when passed
         'group': directives.flag,           # group paths by tag when passed
+        'format': str,                      # "rst" (default) or "markdown"
```

Comment:
I wonder, can we specify here a function that will validate that a format is either "rst" or "markdown"?